### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by humleflue**

### CODE REVIEW

File: ANTLR/ATConditionalSelection.buff [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    return false;
+    end
+
+boolean isNegative(number x) = NOT isPositive(x);
+
+isPositive(-10)?;
+isPositive(10)?;
+
+isNegative(-10)?;
+isNegative(10)?;
```
Comment (JacobToftgaardRasmussen) [inline ANTLR/ATConditionalSelection.buff]: Missing new line at EOF

File: ANTLR/ATLoopingConstruct.buff [hunk 2]
```diff
@@ -4,6 +4,4 @@ number fact(number x) =
     return fact(x - 1)? * x;
     end
 
-fact(10)?;
-
-number foo() = if(true) return 1; return 0; end
\ No newline at end of file
+fact(10)?;
```
Comment (JacobToftgaardRasmussen) [inline ANTLR/ATLoopingConstruct.buff]: Missing new line at EOF

File: ANTLR/ATSequentialOperation.buff [hunk 3]
```diff
@@ -0,0 +1,4 @@
+number plus(number a, number b) = a + b;
+
+plus(1, 2)?;
+plus(-5, 6)?;
```
Comment (JacobToftgaardRasmussen) [inline ANTLR/ATSequentialOperation.buff]: Missing new line at EOF

**Review by JacobToftgaardRasmussen — COMMENTED** [review]
Please move these tests into the Test directory as well

**Review by laurits-mumberg — APPROVED** [review]
Jeg synes det er gode og let forståelige eksempler 👍 💯 🥇

**Pull request merged**